test/go/src/tests/api-v200_test.go

```diff
@@ -0,0 +1,220 @@
+package tests
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/ghodss/yaml"
+	"github.com/santhosh-tekuri/jsonschema"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+// Users struct which contains
```

Comment:
What do we mean by "users struct"?

Reply:
oops, cut and paste error, will fix it.

Reply:
all good now, differenence in how Mac and linux handle file names.